source/configuration/data-access-request-service.md

```diff
@@ -61,12 +61,78 @@ Access to data can be controlled as a single entity or at the node level. Contro
-export const NODE_LEVEL_ACCESS = true;
-export const NODE_LABEL = 'Study Arm(s)';
-```
\ No newline at end of file
+
+window.injectedEnv = {
+	NODE_LEVEL_ACCESS: true,
+	NODE_LABEL: 'Study Arm(s)',
+}
+```
+> Note: NODE_LABEL is just a label property and does not restrict users on any specific node. The configuration for that needs to be done in the backend at the time of project initialization. 
+
+
+## UI Configuration(s)
+**File Location:** `$(src)/bento-frontend/src/bento/requestAccessData.js`
+
+This is a configuration file for the Data Access Request page. Custodians can use this file to make changes to the page's look and feel.
+
+### Usage
+To use this configuration, follow these steps:
+
+1. Import the pageTitle and formFields variables from this file into your code.
+2. Implement the DataAccessRequest component in your code using the formFields configuration object.
+3. Customize the page's look and feel by making changes to the formFields object.
+
+### Configuration
+The configuration for the Data Access Request page includes the following variables:
+
+#### `pageTitle`
+This sets the title of the page. By default, it is set to 'Data Access Request'.
+
+#### `aclAPIOptionsField (optional)`
+This is a string that specifies the name of the API field from which the Access Levels Controls (ACL) should be read. If this field is present, the program will read the ACL from the API instead of from the options property of the armIDs field in the formFields array. By default, this is set to 'listArms'.
+
+#### `formFields`
+This is an array of objects that define the fields in the Data Access Request form. Each object has the following properties:
+
+* **id**: a unique identifier for the field.
+* **type**: the type of the input field. Currently supported types are **'textBox'** and **'aclDropdown'**.
+* **required**: a boolean indicating whether the field is required.
+* **label**: the label for the field.
+* **placeHolder**: the placeholder text for the field.
+* **display**: a boolean indicating whether the field should be displayed on the form.
+* **optionsAPIField**: (optional) a string indicating the name of the API field that contains the options for an ACL dropdown field. If this field is not present, the options will be read from the options property.
+* **options**: an array of objects that define the options for an ACL dropdown field. Each object has the following properties:
+    * **id**: a unique identifier for the option.
+    * **name**: the name of the option.
+
+If **aclAPIOptionsField** is not present, the program will read the ACL options from the **options** property of the **armIDs** field in the **formFields** array.
+
+### Customization
+To customize the Data Access Request page, you can make changes to the **formFields** object. You can add, remove, or modify fields as needed. For example, you could add a new field for the user's email address by adding the following object to the **formFields** array:
```

Comment:
Replace with: "To customize the Data Access Request page, changes can be made to the **formFields** object. Fields can be added, removed, or modified as needed. For example, a new field for the user's email address could be displayed by adding the following object to the **formFields** array:

Reply:
Changes Done.